include/canPowertrain.h

```diff
@@ -4,6 +4,13 @@
 #include "canmanager.h"
 #include "IOManagement.h"
 
+#define I_OUT_12V_TELEM_ID 0x500
+#define V_OUT_12V_TELEM_ID 0x501
+#define SUPP_I_TELEM_ID 0x502
+#define BATT_I_TELEM_ID 0x503
+#define SUPP_V_TELEM_ID 0x504
+#define POWERTRAIN_BOOLS 0x505
```

Comment:
These macros aren't super necessary since they're only used in `sendPowertrainData()`. You can write the IDs in that function or keep it as is, either is fine with me. Just wanted to clarify the use of macros.